documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+# Breaking Change Policy
+
+Customers and MVPs have told us many times in emphatic terms that breaking changes are a continuing problem in Azure PowerShell releases.  These changes cause an undue burden on users maintaining scripts to manage their Azure assets, and this causes reluctance in picking up the later versions of Azure PowerShell. It is clear that continuing, unmanaged breaking changes are unnacceptable to customers. 
+
+**To address customer concerns, Azure PowerShell is committing to limit breaking changes to one release per year and to treat any unplanned breaking change that ships as 
```

Comment:
I have concerns about this, it's going to significantly limit module authors flexibility and it doesn't dramatically improve the end user experience.  Instead I would suggest that every breaking change requires a 6 month deprecation period (with warnings, etc) but that the deprecation period can start at any time.